File: projet pi smart moby/src/main/java/org/example/models/Produit.java
package org.example.models;

public class Produit {
    private int id_produit;
    private String nom;
    private String type;
    private double prix;

    public Produit(int id_produit, String nom, String type, double prix) {
        this.id_produit = id_produit;
        this.nom = nom;
        this.type = type;
        this.prix = prix;
    }

    public int getId() {
        return id_produit;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }

    @Override
    public String toString() {
        return id_produit + " -- " + nom + "--" + type + " -- " + prix + "€";
    }
}

File: src/Entity/Produit.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;


#[ORM\Entity]
class Produit
{

    #[ORM\Id]
    #[ORM\Column(type: "integer")]
    private int $idproduit;

    #[ORM\Column(type: "string", length: 100)]
    private string $nom;

    #[ORM\Column(type: "string", length: 100)]
    private string $type;

    #[ORM\Column(type: "integer")]
    private int $prix;

    public function getIdproduit()
    {
        return $this->idproduit;
    }

    public function setIdproduit($value)
    {
        $this->idproduit = $value;
    }

    public function getNom()
    {
        return $this->nom;
    }

    public function setNom($value)
    {
        $this->nom = $value;
    }

    public function getType()
    {
        return $this->type;
    }

    public function setType($value)
    {
        $this->type = $value;
    }

    public function getPrix()
    {
        return $this->prix;
    }

    public function setPrix($value)
    {
        $this->prix = $value;
    }
}
